asserts/system_user.go

```diff
@@ -216,6 +230,18 @@ func checkHashedPassword(headers map[string]interface{}, name string) (string, e
 	return pw, nil
 }
 
+func checkSystemUserExpiration(headers map[string]interface{}) (string, error) {
+	str, err := checkOptionalString(headers, "user-presence")
+	if err != nil || str == "" {
+		return "", err
+	}
+
+	if str == "until-expiration" {
+		return str, nil
+	}
+	return "", fmt.Errorf("cannot parse 'user-presence': %q is invalid", str)
```

Comment:
invalid "user-presence" header, only explicit valid value is "until-expiration":  %q

Reply:
Good point! Updated  this